projects/ngx-braintree/src/lib/ngx-braintree.component.ts

```diff
@@ -184,10 +189,16 @@ export class NgxBraintreeComponent implements OnInit {
         this.configureDropinService.configureThreeDSecure(this.dropinConfig, this.chargeAmount);
       }
 
-      braintree.dropin.create(this.dropinConfig, (createErr, instance) => {
+      braintree.dropin.create(this.dropinConfig, (createErr: Error, instance) => {
         if (createErr) {
-          console.error(createErr);
-          this.errorMessage = createErr;
+          if (
+            !this.isDestroyed
+            || createErr.message !== 'Selector does not reference a valid DOM node.'
```

Comment:
@petr-vesely please add brackets so its more clear how the condition was exactly meant